Basic/2DSpaceShooter/README.md

```diff
@@ -1,14 +1,31 @@
-## 2D Space Shooter
+# 2D Space Shooter
 
-This is a UNet sample project converted to Netcode for GameObjects.
+![UnityVersion](https://img.shields.io/badge/Unity%20Version:-2021.3%20LTS-57b9d3.svg?logo=unity&color=2196F3)
+![NetcodeVersion](https://img.shields.io/badge/Netcode%20Version:-1.2.0-57b9d3.svg?logo=unity&color=2196F3)
+[![LatestRelease](https://img.shields.io/badge/Latest%20%20Github%20Release:-v1.2.0-57b9d3.svg?logo=github&color=brightgreen)](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/releases/tag/v1.2.0)
+<br><br>
 
-## Sample Features
+This is a UNet sample project converted to Netcode for GameObjects. The 2DSpaceShooter sample is a bitesize sample designed to demonstrate networked 2D and physics-based character movement. 
+<br><br>
 
-This sample shows you:
+# Sample Overview
 
-- Server authorative physics based movement
-- Managing health and a list of buffs for your players in a multiplayer game
+In this sample, learn more about:
+
+- Server authorative physics based movement using Netcode for GameObject's `NetworkRigidbody2D` component
+- Managing health and a list of buffs for your players in a multiplayer game with a `NetworkVariable`
 - How to pool network objects such as bullets and asteroids to improve performance
```

Comment:
Perhaps in we could also add a sentence or two to expand on _why_ these things are useful? Just thinking about learners on the more intermediate end coming to these samples? As in something like:

Why is this useful?

**Object Pooling**
When an object is created, it requires memory allocation and initialisation. When an object is destroyed, it also needs to release its allocated resources. This performance cost can become significant in games where objects are created and destroyed frequently. Pooling network objects can help reduce the overhead by reusing existing objects that have already been initialised and are still in memory. 

Let me know what you think... don't want to bloat the readmes though so could otherwise/also link to some docs or something?

Reply:
this does seem like something that could be linked to in a doc actually, though I know that we're still discussing what should be in our readmes vs our doc with Larah so let's see what Larah says!

Reply:
this is more of a theory explanation as well so let's save this for the docs!